.github/workflows/ci.yml

```diff
@@ -7,15 +7,17 @@ on:
       - '!master'
 
 env:
-  BUILDER_VERSION: v0.3.1
+  BUILDER_VERSION: v0.5.9
```

Comment:
v0.6.0

Reply:
I went to v0.0.6 but had to go back to v0.5.9. Seemed like some infinite recursion going on:

https://github.com/awslabs/aws-c-compression/runs/512841679
```
...
  File "<frozen importlib._bootstrap>", line 1007, in _handle_fromlist
RecursionError: maximum recursion depth exceeded in comparison
    Found project: aws-c-compression at /root/aws-c-compression
    Found project: aws-c-compression at /root/aws-c-compression
    Found project: aws-c-compression at /root/aws-c-compression
    Found project: aws-c-compression at /root/aws-c-compression
    Found project: aws-c-compression at /root/aws-c-compression
    Found project: aws-c-compression at /root/aws-c-compression
    Found project: aws-c-compression at /root/aws-c-compression
    Found project: aws-c-compression at /root/aws-c-compression
    Found project: aws-c-compression at /root/aws-c-compression
...
```